k8s/testnet.yaml

```diff
@@ -1,61 +1,43 @@
 kind: Service
 apiVersion: v1
 metadata:
-  name: testnet-svc
+  name: chainlink-svc
 spec:
   selector:
-    app: testnet
+    app: chainlink
   ports:
     - name: cl
       protocol: TCP
       port: 6688
-      targetPort: 6688
-    - name: rpc
-      protocol: TCP
-      port: 8545
-      targetPort: 8545
-    - name: ws
-      protocol: TCP
-      port: 8546
-      targetPort: 8546
   type: LoadBalancer
 ---
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: pv-claim
+  name: chainlink-pv-claim
 spec:
   accessModes:
     - ReadWriteOnce
   resources:
     requests:
-      storage: 100Gi
+      storage: 16Gi
```

Comment:
My intuition tells me that both parity and chainlink will want more than 16gigs. No one likes going down because of disk space, and it's so cheap, why not get a bunch? 💾 💾 💾